include/tvm/operation.h

```diff
@@ -185,21 +185,42 @@ class PlaceholderOpNode : public OperationNode {
 /*!
  * \brief A Compute op that compute a tensor on certain domain.
```

Comment:
stressing that it is the base class of `ComputeOp` and `TensorComputeOp`

Reply:
done.